src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class ParserTest {
+    Ui ui = new Ui();
+    TaskList tasks = new TaskList();
+    Storage storage = new Storage("data/duke.txt", tasks);
+    @Test
+    public void checkTodo() throws DukeException {
```

Comment:
It might be better to name your test methods in the format of: `<methodName>Test_<inputDescription>_<expectedOutcome>`.
For example, `parserConstructorTest_toDoTask_success`
What do you think?

Reply:
Wow interesting, I've never known test is supposed to be written like that. But it makes sense and makes it more descriptive. Thanks a lot!